react-app/src/app/screens/Game/index.js

```diff
@@ -27,23 +27,32 @@ class Game extends Component {
 
   createHistoryButton(move) {
```

Comment:
I would extract a component called `Moves` that contains the history management logic